sql/core/src/test/resources/sql-tests/results/postgreSQL/window_part3.sql.out

```diff
@@ -391,7 +391,7 @@ SELECT nth_value(four, 0) OVER (ORDER BY ten), ten, four FROM tenk1
 struct<>
 -- !query output
 org.apache.spark.sql.AnalysisException
-cannot resolve 'nthvalue(spark_catalog.default.tenk1.`four`, 0)' due to data type mismatch: The 'offset' argument of nth_value must be greater than zero but it is 0.; line 1 pos 7
+cannot resolve 'nth_value(spark_catalog.default.tenk1.`four`, 0)()' due to data type mismatch: The 'offset' argument of nth_value must be greater than zero but it is 0.; line 1 pos 7
```

Comment:
Do you know why we have an extra `()`?

Reply:
Because the code in `Expression`
```
  def sql: String = {
    val childrenSQL = children.map(_.sql).mkString(", ")
    s"$prettyName($childrenSQL)"
  }
```

Reply:
Can we fix it?

Reply:
https://github.com/apache/spark/pull/29886#discussion_r495697154 fixes it